dcos/marathon.py

```diff
@@ -54,7 +54,11 @@ def _to_error(response):
     if isinstance(response, Error):
         return DefaultError(_default_marathon_error(response.error()))
 
-    message = response.json().get('message')
+    try:
+        message = response.json().get('message')
+    except ValueError as e:
```

Comment:
filed a github issue. 

Why should we check the response code? In this case it would be 400, and we could just say `Bad Request`, but isn't it more helpful for the user that they know what the error is (ie bad json)?